source/reporters/ut_realtime_reporter.tpb

```diff
@@ -57,25 +57,27 @@ create or replace type body ut_realtime_reporter is
       self.print_node('name', a_suite.name);
       self.print_node('description', a_suite.description);
       <<suite_elements>>
+      self.print_start_node('items');
       for i in 1 .. a_suite.items.count loop
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Replace this FOR loop by a more robust WHILE one. ([plsql:ForLoopUsageCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AForLoopUsageCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=819&issues=AWhDYNbDXONnvcWDUk4S&open=AWhDYNbDXONnvcWDUk4S)

[ISSUE_KEY]: AWhDYNbDXONnvcWDUk4S